ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml

```diff
@@ -0,0 +1,127 @@
+base_images:
+  go-builder-base:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  web-builder-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  use_build_cache: true
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    go-builder-base:
+      as:
+      - docker.io/library/golang:1.21
+    web-builder-base:
+      as:
+      - docker.io/library/node:16-alpine
+  to: network-observability-console-plugin
+- build_args:
+  - name: BUILDSCRIPT
+    value: :standalone
+  dockerfile_literal: |
+    ARG BUILDSCRIPT=
+    ARG LDFLAGS=
+    ARG TARGETPLATFORM=linux/amd64
+    ARG BUILDPLATFORM=linux/amd64
+    FROM --platform=$BUILDPLATFORM docker.io/library/node:16-alpine as web-builder
+    USER node
+
+    ARG BUILDSCRIPT
+    ARG TARGETPLATFORM
+    ARG TARGETARCH=amd64
+    WORKDIR /opt/app-root
+
+    COPY --chown=node web/package.json web/package.json
+    COPY --chown=node web/package-lock.json web/package-lock.json
+    WORKDIR /opt/app-root/web
+    RUN npm ci
+
+    WORKDIR /opt/app-root
+    COPY --chown=node web web
+    COPY mocks mocks
+
+    WORKDIR /opt/app-root/web
+    RUN npm run format-all
+    RUN npm run build$BUILDSCRIPT
+
+    FROM --platform=$BUILDPLATFORM docker.io/library/golang:1.21 as go-builder
+
+    ARG TARGETPLATFORM
+    ARG TARGETARCH=amd64
+    ARG LDFLAGS
+    WORKDIR /opt/app-root
+
+    COPY go.mod go.mod
+    COPY go.sum go.sum
+    COPY vendor/ vendor/
+    COPY .mk/ .mk/
+    COPY cmd/ cmd/
+    COPY pkg/ pkg/
+
+    RUN CGO_ENABLED=0 GOARCH=$TARGETARCH go build -ldflags "$LDFLAGS" -mod vendor -o plugin-backend cmd/plugin-backend.go
+
+    FROM  --platform=$TARGETPLATFORM registry.access.redhat.com/ubi9/ubi-minimal:9.3
+
+    COPY --from=web-builder /opt/app-root/web/dist ./web/dist
+    COPY --from=go-builder /opt/app-root/plugin-backend ./
+
+    FROM cypress/included:13.6.5
+    RUN whoami
+    WORKDIR /opt/app-root
+    COPY --from=web-builder /opt/app-root/web/ ./web
+    COPY --from=go-builder /opt/app-root/plugin-backend ./
+
+    COPY mocks mocks
+    COPY config/sample-config.yaml ./config/config.yaml
+    RUN sed -i -e 's/useMocks: false/useMocks: true/g' ./config/config.yaml
+    COPY scripts/cypress.sh .
+    RUN mkdir -p /.config && chmod 775 -R /.config
+    RUN chmod -R 775 ./cypress.sh && chgrp -R 0 /opt/app-root && \
+        chmod -R g+rwX /opt/app-root /root web
```

Comment:
```suggestion
    RUN mkdir -p /home/user && chmod -R 775 ./cypress.sh && \
        chgrp -R 0 /opt/app-root /home/user && \
        chmod -R g+rwX /opt/app-root /root web /home/user
    ENV HOME=/home/user
```

Reply:
wouldn't random UID have its own user directory generated? how would it use /home/user directory?

Reply:
it's enough to set the folder's group and the group permissions